sdk/examples/client/output/expiration.rs

```diff
@@ -67,8 +68,10 @@ async fn main() -> Result<()> {
         .finish()
         .await?;
 
-    println!("Transaction sent: {node_url}/api/core/v2/blocks/{}", block.id());
-    println!("Block metadata: {node_url}/api/core/v2/blocks/{}/metadata", block.id());
+    println!(
+        "Block with ExpirationUnlockCondition sent: {explorer_url}/block/{}",
```

Comment:
Sounds a bit wrong since the unlock condition is only for the output and not the block